index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <meta  charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+    <title>Form-basic</title>
+
+  </head>
+
+  <body>
+    <!-- Top level container -->
+    <div class="container">
+
+      <!-- PAGE HEADER -->
+      <DIV  class="header" >
+        <h1 id="title" class="text-center">HAPPY SURVEY</h1>
+        <p id="description" class="description text-center">
+          Another happy survey, we really want to hear about you!!
+        </p>
+      </DIV>
+
+      <!-- PAGE FORM -->
+      <form id="survey-form" action="mailto://jpolo.santamaria@gmail.com">
+
+        <div class="form-group">
+          <label id="name-label" for="name">Whats your Name</label>
+          <input
+            type="text"
+            name="name"
+            id="name"
+            class="form-control"
+            placeholder="write your name Here"
+            required
+          />
+        </div>
+
+        <div class="form-group">
+          <label id="email-label" for="email">Email</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            class="form-control"
+            placeholder="Please your email goes here"
+            required
+          />
+        </div>
+
+        <div class="form-group">
+          <label id="number-label" for="number">Age<span class="clue">(optional)</span></label>
+          <input
+            type="number"
+            name="age"
+            id="number"
+            min="10"
+            max="99"
+            class="form-control"
+            placeholder="Your age goes here "/>
+        </div>
+
+        <div class="form-group">
+          <SPAN >How do you get your income? </SPAN>
```

Comment:
Here's another example of a tag that has been used incorrectly. Please update your project to use all HTML tags in lower case.